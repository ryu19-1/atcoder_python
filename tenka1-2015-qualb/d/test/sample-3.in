25
10
BED
BEL
BIG
DIE
DIG
DOG
OLD
shE
qED
ZOO
