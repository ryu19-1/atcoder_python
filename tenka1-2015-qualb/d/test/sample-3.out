918910900738345310070038
